src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import { Draggable } from 'react-beautiful-dnd';
+import { FC, useContext } from 'react';
+import { draggableProps } from '@/interfaces/availabilityPanel.type';
+import classNames from '@/components/availability-panel/drag-drop-context/styles.module.scss';
+import { disableDrag } from './index';
+
+const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
+
+const getItemStyle = (
+  isDragging: boolean,
+  draggableStyle: any,
+  draggableId1: string,
+  draggableId2: string,
+  draggableId: string,
+) => {
+  const style = {
+    // eslint-disable-next-line no-nested-ternary
+    background: draggableId1 === draggableId || draggableId2 === draggableId ? '#aeaeae' : isDragging ? '#d1d1d1' : 'white',
```

Comment:
than doing this, let's please use simple if else statements
lets not compromise on readability of code for length of code